*/test/message/events_unhandled_error_sameline.js:4
new EventEmitter().emit('error', new Error());
                                 ^

Error
    at Object.<anonymous> (*events_unhandled_error_sameline.js:*:*)
    at Module._compile (internal/modules/cjs/loader.js:*:*)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:*:*)
    at Module.load (internal/modules/cjs/loader.js:*:*)
    at Function.Module._load (internal/modules/cjs/loader.js:*:*)
    at ModuleWrap.<anonymous> (internal/modules/esm/translators.js:*:*)
    at ModuleJob.run (internal/modules/esm/module_job.js:*:*)
    at async Loader.import (internal/modules/esm/loader.js:*:*)
Emitted 'error' event at:
    at Object.<anonymous> (*events_unhandled_error_sameline.js:*:*)
    at Module._compile (internal/modules/cjs/loader.js:*:*)
    [... lines matching original stack trace ...]
    at async Loader.import (internal/modules/esm/loader.js:*:*)
